webapp/channels/src/packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -153,20 +154,15 @@ export function getPost(postId: string): ActionFuncAsync<Post> {
 
         try {
             post = await Client4.getPost(postId);
-            getMentionsAndStatusesForPosts([post], dispatch, getState);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
             dispatch({type: PostTypes.GET_POSTS_FAILURE, error});
             dispatch(logError(error));
             return {error};
         }
 
-        dispatch(batchActions([
-            receivedPost(post, crtEnabled),
-            {
-                type: PostTypes.GET_POSTS_SUCCESS,
-            },
-        ]));
+        dispatch(receivedPost(post, crtEnabled));
```

Comment:
Do we not need to batch these actions anymore?

Reply:
They can't be batched any more because you can only batch non-thunk actions like the one created by `receivedPost`